charts/prefect-server/values.yaml

```diff
@@ -54,6 +54,8 @@ server:
     prefectUiApiUrl: ""
     # -- sets PREFECT_UI_URL
     prefectUiUrl: ""
+    # -- sets PREFECT_UI_STATIC_DIRECTORY
+    prefectUiStaticDirectory: "/ui_build"
```

Comment:
this needs a default value, or else the server container will look for a non-existent default that is baked into the application code.  so the expectation is, when using the helm chart, we will be setting this by either user-provided value or this default value